test/common/upstream/eds_test.cc

```diff
@@ -497,8 +507,9 @@ TEST_F(EdsTest, EndpointRemoval) {
 
   add_endpoint(80);
   add_endpoint(81);
-
-  VERBOSE_EXPECT_NO_THROW(cluster_->onConfigUpdate(resources, ""));
+  resources_.Clear();
```

Comment:
Nit: I prefer just using brace scoped variables for resources rather than globals that we keep adding/mutating; unless there is some other reason we need them to be globals (e.g. mock setup etc)

Reply:
Yeah, definitely, what I had was fairly gross. I think I have a pretty clean approach now.